.github/workflows/generate.yml

```diff
@@ -65,20 +91,29 @@ jobs:
       - name: GCS Set cache-control
         run: |
           gsutil setmeta -h "Cache-Control: no-store" gs://${{ github.event.inputs.network }}-cdn/networks/${{ github.event.inputs.network }}-${{ github.event.inputs.namespace }}/*
-
       - name: Verify genesis time
         run: |
           wget -O ./network-config-test.yaml https://storage.googleapis.com/${{ github.event.inputs.network }}-cdn/networks/${{ github.event.inputs.network }}-${{ github.event.inputs.namespace }}/network-config.yaml?ignoreCache=1
           echo $(sha256sum ./network-config-test.yaml)
           echo $SHA256_SUM
-
-      - name: Request l15 network upgrade
-        if: ${{ github.event.inputs.requestUpgrade == 'true' }}
+      - name: Request l15-dev network upgrade
+        if: ${{ github.event.inputs.requestUpgrade == 'true' && github.event.inputs.timestampMethod == 'timestamp' }}
         env:
           GITHUB_TOKEN: ${{ secrets.GH_DISPATCH_TOKEN }}
         run: |
           curl -X POST \
           -H "Accept: application/vnd.github.v3+json" \
           -d '{"event_type":"l15_restart_trigger", "client_payload": {"repository": "network-configs", "namespace": "${{ github.event.inputs.namespace }}"}}' \
           -u "$GITHUB_TOKEN" \
-          https://api.github.com/repos/lukso-network/network-gitops/dispatches
+          https://api.github.com/repos/lukso-network/network-helm-charts/dispatches          
```

Comment:
We are using `https://api.github.com/repos/lukso-network/network-gitops/dispatches` now.